vllm/utils/flashinfer.py

```diff
@@ -144,22 +144,41 @@ def has_nvidia_artifactory() -> bool:
         return False
 
 
+def support_trtllm_attention(
+    num_qo_heads: int,
+    num_kv_heads: int,
+) -> bool:
+    """
+    Return ``True`` if the attention satisfies the minimum requirement
+    of TRT-LLM kernel
+    """
+    # Requires SM100 and NVIDIA artifactory to be accessible to download cubins
+    # and check if the dimensions are supported
+    return (current_platform.is_device_capability(100)
+            and has_nvidia_artifactory() and num_qo_heads % num_kv_heads == 0)
```

Comment:
not related to this PR, but I think `has_nvidia_artifactory()` can be removed because FlashInfer now supports downloading all cubins at once.

Reply:
can do this in another PR